src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -856,7 +858,7 @@ MulticopterPositionControl::task_main_trampoline(int argc, char *argv[])
 void
 MulticopterPositionControl::update_ref()
 {
-	if (_local_pos.ref_timestamp != _ref_timestamp) {
+	if (_local_pos.ref_timestamp != _ref_timestamp && _first_origin_set) {
```

Comment:
@priseborough This PR just works for the situation where we use flow at the beginning and then gain GPS. The reason is that this line will never be fully true. If `_first_origin_set` is true, `_ref_timestamp` has been set to `_local_pos.ref_timestamp`. Hence we will never enter this code block.
For the normal simulation (`make posix_sitl_default gazebo_iris`) the quad climbs into the sky when you run `commander takeoff`.

Reply:
OK, I will review and fix tomorrow.